src/models/CookieConsent.js

```diff
@@ -66,10 +66,10 @@ export default class CookieConsent extends Base {
     return this.popup.consents;
   }
 
-  deleteCookie(name) {
-    const { domain, path, secure } = this.popup.options.cookie;
+  deleteCookie(name, attributes) {
```

Comment:
This makes the parameter optional.

```suggestion
  deleteCookie(name, attributes = {}) {
```